docs/store-operations/shipping/msf-international-enhancements.mdx

```diff
@@ -0,0 +1,75 @@
+# Shipping - International Enhancements for Multi-Storefront
+
+When you sell internationally or to multiple customer segments, you may want the available shipping methods for a shipping zone to depend on the shopper's region or segment.
```

Comment:
```suggestion
When you sell internationally to multiple customer segments, you may want the available shipping methods for a shipping zone to depend on the shopper's region or segment.
```

Reply:
I think it's actually 2 distinct use cases (multi-region & multi-segment)

Reply:
Oh okay, I wasn't reading that right.